packages/astro/README.md

```diff
@@ -21,3 +21,33 @@ npm init astro
 ```
 
 ### [Full Documentation Site →](https://docs.astro.build/)
+
+## Sponsors
```

Comment:
Perfect for now. In the future, I really like the SVG that antfu uses here: https://github.com/antfu/unplugin-icons#sponsors. Especially since we have many repos, it would be easier to just reference a single, central image that we could re-build every 24 hours (like https://github.com/antfu/static)

Reply:
Yeah I like that SVG too. Just wanted to throw something in the README for now but I’d be +1 on making something fancier (we could probably even keep an SVG in the GitHub but designing it/making it would take me some time).